src/app/client/src/app/plugins/profile/components/delete-account/delete-account.component.ts

```diff
@@ -113,8 +114,24 @@ export class DeleteAccountComponent implements OnInit, OnDestroy {
   }
 
   verificationSuccess(data) {
-      window.location.replace('/logoff');
-      this.cacheService.removeAll();
+      this.userService.deleteUser().subscribe(
+        (data: ServerResponse) => {
+          if(_.get(data, 'result.response') === 'SUCCESS'){
+            window.location.replace('/logoff');
+            this.cacheService.removeAll();
+            if(this.deviceDetectorService.isMobile()){
+              const url ='dev.sunbird.app://mobile?userId'+ this.userProfile.userId;
```

Comment:
It is hardcoded, It should take it from env variable.

attach action & additional info to the env variable so hat later for any actions, we have use the same deeplink.

Reply:
this will be taken up post discussion